01-connecting-to-simulator.py

```diff
@@ -0,0 +1,19 @@
+  
+#!/usr/bin/env python3
+#
+# Copyright (c) 2019 LG Electronics, Inc.
+#
+# This software contains code licensed as described in LICENSE.
+#
+
+import simulation.config as config
+
+# Connects to the simulator instance at the ip defined by SIMULATOR_HOST, default is localhost or 127.0.0.1
+cf = config.Config()
+sim = cf.Simulator()
+
+# sim = lgsvl.Simulator(os.environ.get("SIMULATOR_HOST", "127.0.0.1"), 8181)
```

Comment:
why we comment out this line?

Reply:
Deleted. 
sim = cf.Simulator() already do the same job for this line of code.